public/pages/utils/helpers.js

```diff
@@ -68,6 +72,18 @@ export const appendCommentsAction = (columns, httpClient) => {
   return columns;
 };
 
+export async function getIsAgentConfigured(dataSourceId){
+  const assistantClient = getAssistantClient();
+  const res = await assistantClient.agentConfigExists(
+    [
+      SUMMARY_AGENT_CONFIG_ID,
+      LOG_PATTERN_SUMMARY_AGENT_CONFIG_ID,
+    ],
+    { dataSourceId: dataSourceId }
+  );
+  return res.exists;
```

Comment:
Can you also handle the error case to avoid throwing exception to corrupt the following rendering?

Reply:
Sure, have added a handler to return false instead of throwing exception